lib/rps_game.rb

```diff
@@ -0,0 +1,29 @@
+class RPSGame
+  
+  attr_reader :player_1_choice, :player_2_choice
+ 
+  def initialize(player_1_choice, player_2_choice = ComputerChoice.new.choice)
+    @player_1_choice = player_1_choice
+    @player_2_choice = player_2_choice
+  end
+
+  def result
+    if player_1_choice == player_2_choice
+      'Draw'
+    else
+      player_win?
+    end
+  end
+
+  private
+
```

Comment:
could you extent the game to rock paper scissors lizard spock using similar logic?